doc/news/changes/incompatibilities/20170807Jean-PaulPelteret

```diff
@@ -0,0 +1,4 @@
+Changed: The boundary_id and material_id types have been changed from
```

Comment:
can you write this as `types::boundary_id` and `types::material_id` so that doxygen properly links to it?